<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>359</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>593</width>
    <height>359</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>593</width>
    <height>359</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="buttonGetPaths">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>20</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>refresh</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboPaths">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font-size: 14pt;</string>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>571</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="buttonUp">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonSuspend">
       <property name="text">
        <string>suspend</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonReload">
       <property name="text">
        <string>reload</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="buttonHalt">
       <property name="text">
        <string>halt</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTextEdit" name="textEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>571</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>buttonHalt</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
